kbs/src/api/src/attestation/coco/grpc.rs

```diff
@@ -85,10 +89,27 @@ impl Attest for Grpc {
     }
 
     async fn verify(&mut self, tee: Tee, nonce: &str, attestation: &str) -> Result<String> {
+        let attestation: Attestation = serde_json::from_str(attestation)?;
+        let runtime_data = vec![
+            STANDARD.encode(nonce.as_bytes()),
+            STANDARD.encode(attestation.tee_pubkey.k_mod.as_bytes()),
+            STANDARD.encode(attestation.tee_pubkey.k_exp.as_bytes()),
+        ];
+
+        let pubkey_claim = json!({
+            "tee-pubkey": attestation.tee_pubkey,
+        });
+
+        let customized_claims = serde_json::to_string(&pubkey_claim)?;
+
         let req = tonic::Request::new(AttestationRequest {
-            tee: to_grpc_tee(tee) as i32,
-            nonce: String::from(nonce),
-            evidence: String::from(attestation),
+            tee: to_grpc_tee(tee).into(),
+            evidence: STANDARD.encode(attestation.tee_evidence),
+            runtime_data,
+            init_data: vec![],
```

Comment:
cc @mkulke You might be interested in this -- This is the KBS side code that does not leverage the ability of the initdata verafication in this PR.